static/templates/settings/edit_custom_profile_field_form.hbs

```diff
@@ -36,6 +36,13 @@
         <input type="url" name="url_pattern" autocomplete="off" maxlength="80" />
     </div>
     {{/if}}
+    <div class="input-group">
+        <label class="checkbox profile_field_display_label" for="edit_editable_by_user_{{id}}">
+            <input type="checkbox" id="edit_editable_by_user_{{id}}"name="editable_by_user" {{#if editable_by_user}} checked="checked" {{/if}}/>
+            <span></span>
+            {{t 'User can edit this field' }}
```

Comment:
This should be "Users", as per the issue desciption.

Reply:
I am sorry for this typo will make changes.